khmer/kfile.py

```diff
@@ -123,3 +125,17 @@ def check_valid_file_exists(in_files):
         else:
             print >>sys.stderr, 'WARNING: Input file %s not found' % \
                                 in_file
+
+
+def add_output_compression_type(parser):
+    parser.add_argument('--gzip', default=False, action='store_true',
+                        help='Option to output as gzip')
+    parser.add_argument('--bzip', default=False, action='store_true',
+                        help='Option to output as bz2')
+
+
+def enable_output_compression(args):
```

Comment:
I think this can be generalized a bit - right now the output file has to be in args.output, but for some of the scripts I think there will be a need for different output names/files, especially in the scripts that have multiple output files.

So maybe enable_output_compression can take a file handle, and return an fp (what gzip.GzipFile/bz2file.open give) for it?